sub-packages/bionemo-amplify/tests/bionemo/amplify/test_convert.py

```diff
@@ -285,6 +285,7 @@ def test_convert_amplify_120M_smoke(tmp_path):
     io.import_ckpt(module, f"hf://{model_tag}", tmp_path / "nemo_checkpoint")
 
 
+# Enable these by running `pip install -v -U git+https://github.com/facebookresearch/xformers.git@v0.0.29.post1#egg=xformers`
```

Comment:
can you add those instructions to the skipif msg?

Reply:
Done